oak-run-commons/src/test/java/org/apache/jackrabbit/oak/index/indexer/document/flatfile/TraverseAndSortTaskTest.java

```diff
@@ -34,8 +34,7 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Queue;
-import java.util.concurrent.Callable;
-import java.util.concurrent.Phaser;
+import java.util.concurrent.*;
```

Comment:
Please don't use star imports.

Reply:
yep....
need to figure out why intellij always do that for me...

Reply:
found https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports